code/modules/projectiles/guns/rifles.dm

```diff
@@ -643,6 +643,143 @@
 /obj/item/weapon/gun/rifle/m46c/stripped
 	random_spawn_chance = 0//no extra attachies on spawn, still gets its stock though.
 
+
+//-------------------------------------------------------
+// Commanding Officer Shotgun // Its like rainbow six siege
+
+/obj/item/weapon/gun/rifle/xm52
```

Comment:
I know why it's based on /rifle, but if the intent is for it to be a shotgun it might be wise to put it in the shotgun file. Though that's not a strict necessity.